packages/elements/src/components/ino-carousel/ino-carousel.tsx

```diff
@@ -0,0 +1,158 @@
+import { Component, ComponentInterface, Element, Host, Prop, Watch, h } from '@stencil/core';
+import classNames from 'classnames';
+
+
+@Component({
+  tag: 'ino-carousel',
+  styleUrl: 'ino-carousel.scss',
+  shadow: false
+})
+export class InoCarousel implements ComponentInterface{
+
+  @Element() el: HTMLElement;
+  private slides: HTMLInoCarouselSlideElement[];
+  private currentSlide: number = 0;
+  private slideCounter: number = 1;
+  private timer: NodeJS.Timeout;
+
+  /**
+   * Optional group value to manually manage the displayed slide
+   */
+  @Prop() value?: any;
+  @Watch('value')
+  valueChanged(newVal: any) {
+    this.addSlideAnimation(this.currentSlide); // adds the slide animation to the current slide
+    this.slides.forEach((slide) => {
+      slide.inoSelected = newVal === slide.value;
+      if(slide.inoSelected) {
+        this.currentSlide = this.slides.indexOf(slide);
+      }
+    });
+    this.addSlideAnimation(this.currentSlide); // adds the slide animation to the new slide
+  }
+
+  /**
+   * Enables autoplay which causes slides to be changed automatically
+   */
+  @Prop() inoAutoplay: boolean = false;
+
+  /**
+   * Disables the slide animation
+   */
+  @Prop() inoAnimated: boolean = false;
```

Comment:
The animation does not seem to work correctly. Take a look:
![carousel-animation](https://user-images.githubusercontent.com/22963121/95225743-6609bf80-07fc-11eb-8f12-f85f0c5878cc.gif)

Is it working for you?

Reply:
It works fine for me - regardless of the browser. Which one are you using? However, I just noticed that the animation is not working in Google Chrome for some reason...

Reply:
it should work now